website/content/partials/alerts/enterprise-and-hcp.mdx

```diff
@@ -0,0 +1,6 @@
+<EnterpriseAlert product="vault">
+  <a href="https://www.hashicorp.com/products/vault/pricing">Vault Enterprise</a>
+  &nbsp;license or&nbsp;
+  <a href="/hcp/docs/vault/tiers-and-features#plus-tier">HCP Vault Plus</a>
+  &nbsp;cluster required
+</EnterpriseAlert>
```

Comment:
I no longer work for an organization with a Vault Enterprise license so I don't know how handling tiers of license works these days for self-managed Vault Enterprise, but it seems a bit weird to have this `enterprise-and-hcp` partial that mentions Enterprise with no tier qualification, but HCP Vault, Plus tier only.

There were some things in the diff above, where it seems to have been applied to features which don't seem to be documented as requiring **Plus** elsewhere - though I'm not going to pay to check myself.